app/views/additional_details/submissions/_submissions_panel.html.haml

```diff
@@ -40,14 +42,21 @@
         %tbody
           - submissions.each do |submission|
             - service = Service.find(submission.service_id)
+            - ssr = SubServiceRequest.find(submission.line_item.sub_service_request_id)
             %tr{ class: "submission", data: { id: "#{submission.id}", service_id: "#{service.id}" } }
               %td
                 .text-center
-                  = submission.id
+                  = "#{ssr.display_id}"
               %td
                 .text-center
-                  = link_to_if current_identity.is_super_user? || current_identity.is_service_provider?(SubServiceRequest.find(submission.line_item.sub_service_request_id)), "#{service.name}",
+                  = link_to_if current_identity.is_super_user? || current_identity.is_service_provider?(ssr), "#{service.name}",
                     service_additional_details_questionnaires_path(service), target: '_blank'
+              %td
+                .text-center
+                  - if submission.created_at == submission.updated_at
+                    = submission.created_at.strftime("%m/%d/%Y")
+                  - else
+                    = submission.updated_at.to_date("%m/%d/%Y")
```

Comment:
Aren't `created_at` and `updated_at` the same type? Just curious about why different date formatting methods are being used (`strftime` and `to_date`).